src/init.cpp

```diff
@@ -868,6 +878,55 @@ void ThreadImport(std::vector<fs::path> vImportFiles)
     }
 }
 
+void PeriodicStats()
+{
+    CCoinsStats stats;
+    FlushStateToDisk();
+    if (GetUTXOStats(pcoinsdbview.get(), stats)) {
+        statsClient.gauge("utxoset.tx", stats.nTransactions, 1.0f);
+        statsClient.gauge("utxoset.txOutputs", stats.nTransactionOutputs, 1.0f);
+        statsClient.gauge("utxoset.dbSizeBytes", stats.nDiskSize, 1.0f);
+        statsClient.gauge("utxoset.blockHeight", stats.nHeight, 1.0f);
+        statsClient.gauge("utxoset.totalAmount", (double)stats.nTotalAmount / (double)COIN, 1.0f);
+    } else {
+        // smth went wrong
```

Comment:
Proposal to add a linter for `smth` :joy: 
```suggestion
        // something went wrong
```